core/templates/pages/exploration-editor-page/history-tab/history-tab.directive.spec.ts

```diff
@@ -155,5 +156,44 @@ describe('HistoryTab controller', function() {
         expect(historyTabCtrl.versionNumbersToDisplay).toEqual([2, 1]);
       }
     );
+
+    it('Check the box has disabled properly',
+      function() {
+        historyTabCtrl.displayedCurrentPageNumber = 1;
+        historyTabCtrl.selectedVersionsArray = [1];
+        expect(historyTabCtrl.isCheckboxDisabled(31)).toBe(false);
+      }
+    );
+
+    it('testing changeSelectedVersions function',
```

Comment:
Usually the names should be something that can be said as a continuation of 'it...', can you rephrase this to be more descriptive?